masterdata/configuration/liquibase/liquibase.xml

```diff
@@ -323,4 +323,34 @@
             1, now(), NULL, NULL, NULL, uuid());
         </sql>
     </changeSet>
+
+    <changeSet id="BAH-3917-25232" author="standard-config">
```

Comment:
Adding program attributes is possible via CSV, lets use `attributetypes` domain, That would be much easier for implementers to update than Liquibase.
https://github.com/mekomsolutions/openmrs-module-initializer/blob/master/readme/atttypes.md

Reply:
Updated to csv removed the liquibase change set aswell